alkira/resource_alkira_policy_nat_rule_helper.go

```diff
@@ -110,6 +110,24 @@ func expandPolicyNatRuleAction(in *schema.Set) *alkira.NatRuleAction {
 			e.IpType = v
 		}
 
+		if v, ok := actionValue["src_addr_translation_routing_track_prefixes"].([]interface{}); ok {
```

Comment:
I don't see where we set those values in READ function...

Reply:
In the NAT rule read function it looks like we're not accessing any of the fields for match or action - https://github.com/alkiranet/terraform-provider-alkira/blob/73a65dd3bd9d43b12d7582ad3ce8bb52cb226ebb/alkira/resource_alkira_policy_nat_rule.go#L261

Am I misunderstanding something or should we add all the relevant fields in that function?